# Represents a list of tool calls (a full execution plan)
nlp_drone_control/Action[] actions
uint8 replan_mode          # 0 = APPEND, 1 = REPLACE

